### PR TITLE
Update the disclaimer to note that some ASF files don't have correct ASF header

### PR DESCRIPTION
**Pull request by Ouss4**

### CODE REVIEW

**Comment by patacongo** [comment]
This is a trivial change that does not warrant running the PR changes (one line added to a .txt file)

**Comment by btashton** [comment]
I think we should put more text in here explaining that most of the code currently with BSD headers has a code grant from Greg, and we are working on updating the headers in the coming releases and obtaining approvals from other right holders as we can.

File: DISCLAIMER-WIP [hunk 1]
```diff
@@ -11,6 +11,7 @@ a list of known issues the project is currently aware of (note that this list, b
 is likely to be incomplete): 
 
  * Releases may have incomplete licensing conditions.
+ * Some ASF files don't have the correct headers.
```
Comment (Ouss4) [inline DISCLAIMER-WIP:14]:
```suggestion
 * Some ASF files still have BSD headers.  Most of the top contributors have signed an ICLA and we are working on updating the headers and getting approvals from other copyright holders.
```
@btashton I pretty much copied what you wrote.
Please if you get anything to add/remove use the suggestions and we squash the commits when merging.

Comment (btashton) [reply DISCLAIMER-WIP:14]: Looks good to me. I think that makes the state of things clear for people outside the project which I think was Justin's request.

**Comment by btashton** [comment]
This did not get the suggested changed included

**Comment by Ouss4** [comment]
> This did not get the suggested changed included

I submitted a new PR for that.

**Pull request merged**